NOTICE.md

```diff
@@ -11,6 +11,20 @@ The project maintains the following source code repositories:
 
  * https://github.com/eclipse/kiso
 
+## Contributors
+
+Thanks goes to these wonderful people (sorted by last name):
+
+ * Lars Beseke
+ * Michael Bierkandt
+ * Martin Clement
+ * Sebastian Fischer
+ * Mohamed Ali Khalifi
+ * Hans-Uwe Rempler
+ * Yong Shen
+ * Thorsten Udelhoven
```

Comment:
We should confirm that Thorsten having done signing MA and training.

Reply:
He sent me the signed "Erklärung zu Urheberpersönlichkeitsrechten" just today. I stored it in the "MA" network share.